README.md

```diff
@@ -24,6 +25,63 @@ npm run build
 npm run test
 ```
 
+## Live testnet testing
+
+1. Check if the contract deployments repository supports the testnet you will be using. Supported chains can be found [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json). If the testnet is not already supported, proceed to steps 2-4, otherwise you may skip to step 5.
+2. Navigate to the contract deployments repo [here](https://github.com/axelarnetwork/axelar-contract-deployments/) and clone the repository locally.
+3. Within the info folder in the [testnet](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json) file, add the configuration of the new testnet as shown in the example below with correct values:
+
+```json
+{
+    "chains": {
+        "example": {
+            "name": "Example",
+            "id": "example",
+            "chainId": 123,
+            "rpc": "PROVIDER_RPC",
+            "tokenSymbol": "EXM",
+            "gasOptions": {
+                "gasLimit": 8000000
+            }
+        }
+    }
+}
+```
+
+4. In the root directory of this repository navigate to the `hardhat.config.js` file and modify the chains import line as shown below:
+
+```javascript
+const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const chains = require(`[LOCAL_PATH]/axelar-contract-deployments/info/${env}.json`);
+```
+
+5. In the root folder of this repository create a new file named `keys.json`
```

Comment:
does the `keys.json` need to put inside the `info` folder? (I noticed that from the require path in hardhat.config.js)

Reply:
switching to outside info folder in the other PR for consistency

Reply:
good catch!